consensus/reactor.go

```diff
@@ -1375,6 +1381,27 @@ type NewRoundStepMessage struct {
 	LastCommitRound       int
 }
 
+// ValidateBasic performs basic validation.
+func (m *NewRoundStepMessage) ValidateBasic() error {
+	if m.Height < 1 {
+		return errors.New("Negative or zero Height")
```

Comment:
I meant this for everything. Instead of `Height < 0` we might want `Height < 1` everywhere.

Reply:
Let's leave it at `< 0` everywhere for now and revisit.